.github/workflows/ci.yml

```
@@ -0,0 +1,120 @@
+# WARNING: DO NOT EDIT!
+#
+# This file was generated by plugin_template, and is managed by it. Please use
+# './plugin-template --github pulp_file' to update this file.
+#
+# For more info visit https://github.com/pulp/plugin_template
+---
+name: pulp-openapi-generator PR CI
+on: pull_request
```

could you make it to run on push?
```suggestion
on:
  pull_request:
    branches:
      - '*'
  push:
    branches:
      - '*'
```
so we will be able to see the job running

---

I've been making a test PR after the initial one to make sure it's all working properly.  Unless we want it to run on push also permanently?

---

I believe having on push is beneficial because you can see the CI running on your fork before starting a PR, but I agree, splitting into 2 PRs is better